.github/workflows/fork-ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: Fork Docker CI
+
+on:
+  push:
+    branches:
+    - network-operator-*
+    tags:
+    - network-operator-*
+
+jobs:
+  determine_docker_registry_and_tag:
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY_INTERNAL: nvcr.io/nvstaging/mellanox
+      REGISTRY_PUBLIC: nvcr.io/nvidia/cloud-native
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        sparse-checkout: .
+    - if: github.ref_type == 'branch'
+      name: Determine docker registry and tag (when git branch)
+      run: |
+        echo DOCKER_REGISTRY=$REGISTRY_INTERNAL >> $GITHUB_ENV
+        echo DOCKER_TAG=$(git rev-parse --short HEAD) >> $GITHUB_ENV  # short git commit hash
+    - if: github.ref_type == 'tag'
+      name: Determine docker registry and tag (when git tag)
+      run: |
+        echo DOCKER_REGISTRY=$(echo ${{ github.ref_name }} | sed 's/network-operator-//' | grep -q '-' && echo $REGISTRY_INTERNAL || echo $REGISTRY_PUBLIC) >> $GITHUB_ENV
+        echo DOCKER_TAG=${{ github.ref_name }} >> $GITHUB_ENV
+    - name: Store docker registry and tag for following jobs
+      id: store-docker-registry-and-tag
+      run: |
+        echo DOCKER_REGISTRY=$DOCKER_REGISTRY >> $GITHUB_OUTPUT
+        echo DOCKER_TAG=$DOCKER_TAG >> $GITHUB_OUTPUT
+    outputs:
+      docker_registry: ${{ steps.store-docker-registry-and-tag.outputs.DOCKER_REGISTRY }}
+      docker_tag: ${{ steps.store-docker-registry-and-tag.outputs.DOCKER_TAG }}
+
+  build_and_push_images:
+    needs: determine_docker_registry_and_tag
+    runs-on: ubuntu-latest
+    env:
+      BUILD_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le
+      DOCKER_REGISTRY: ${{ needs.determine_docker_registry_and_tag.outputs.docker_registry }}
+      DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
+    strategy:
+      matrix:
+        include:
+        - component: controller
+          image_name: ${{ github.event.repository.name }}
+          dockerfile: Dockerfile
+        - component: config-daemon
+          image_name: ${{ github.event.repository.name }}-config-daemon
+          dockerfile: Dockerfile.sriov-network-config-daemon
+        - component: webhook
+          image_name: ${{ github.event.repository.name }}-webhook
+          dockerfile: Dockerfile.webhook
+    steps:
+    - uses: actions/checkout@v4
+    - uses: docker/setup-qemu-action@v3
```

Comment:
why do we need to specify `docker/setup-qemu-action`?  can we use `docker/setup-buildx-action` only?

Reply:
The [upstream indeed states that it's _optional_](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/.github/workflows/image-push-master.yml#L19) for allowing support of other platforms.
I chose to keep the build steps as similar as possible to the upstream.